x/ante/restricted_tx.go

```diff
@@ -38,6 +40,12 @@ func (d RestrictedTx) AnteHandle(ctx sdk.Context, msgs []sdk.Msg, simulate bool,
 			signer = signers[0]
 		}
 
+		// bypass the permission check if the signer is the governance account. This means the message was sent as a governance proposal,
+		// which is the most restrictive check it can pass.
+		if d.govAccount.Equals(signer) {
+			continue
+		}
+
 		signerRole := d.permission.GetRole(ctx, signer)
 		switch permission.GetPermissionRole((msg).(descriptor.Message)) {
```

Comment:
maybe another approach could have been to add a new permission.GOVERNANCE role and changing the cases to allow the signer who has that role to pass (would need a migration to assign gov module that role I imagine). though the bypass is perfectly fine given our time sensitivity.